pkg/cloud/services/networking/mock_client/client.go

```diff
@@ -0,0 +1,605 @@
+/*
```

Comment:
If we follow CAPA and CAPZ, this file is in pkg/cloud/services/networking/mock_networking.